x-pack/test/accessibility/apps/dashboard_controls.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const a11y = getService('a11y');
+  const testSubjects = getService('testSubjects');
+  const kibanaServer = getService('kibanaServer');
+  const PageObjects = getPageObjects(['common', 'dashboard', 'home']);
+  const browser = getService('browser');
+
+  describe('Dashboard controls a11y tests', () => {
+    before(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+      await PageObjects.common.navigateToUrl('home', '/tutorial_directory/sampleData', {
+        useActualUrl: true,
+      });
+      await PageObjects.home.addSampleDataSet('flights');
+      await PageObjects.common.navigateToApp('dashboard');
+      await testSubjects.click('dashboardListingTitleLink-[Flights]-Global-Flight-Dashboard');
+      await testSubjects.click('dashboardEditMode');
+    });
+
+    after(async () => {
+      await PageObjects.common.navigateToUrl('home', '/tutorial_directory/sampleData', {
+        useActualUrl: true,
+      });
+      await PageObjects.home.removeSampleDataSet('flights');
+    });
+
+    it('Controls main menu panel', async () => {
+      await testSubjects.click('dashboard-controls-menu-button');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Add controls panel', async () => {
+      await testSubjects.click('controls-create-button');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Open data view picker', async () => {
+      await testSubjects.click('open-data-view-picker');
+      await a11y.testAppSnapshot();
+      await browser.pressKeys(browser.keys.ESCAPE);
+    });
+
+    it('Filter by type panel', async () => {
+      await testSubjects.click('toggleFieldFilterButton');
+      await a11y.testAppSnapshot();
+      await testSubjects.click('typeFilter-string');
+    });
+
+    it('Options control panel', async () => {
+      await testSubjects.click('field-picker-select-OriginCityName');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Dashboard with options control panel', async () => {
+      await testSubjects.click('control-editor-save');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Range control panel', async () => {
+      await testSubjects.click('dashboard-controls-menu-button');
+      await testSubjects.click('controls-create-button');
+      await testSubjects.click('field-picker-select-AvgTicketPrice');
+      await a11y.testAppSnapshot();
```

Comment:
Here we take the snapshot while the editor flyout for the range control is still open, while in the test above we take a snapshot after the flyout is closed.

So it looks like we test the dashboard with an options list on it, but never with a range slider on it?